test/cypress/integration/components/header/header.cy.js

```diff
@@ -45,7 +45,7 @@ describe('Header', () => {
       menuDesktop.secondPanel().should('not.be.visible');
     });
 
-    it('clicking from the global search to the mega-menu', () => {
+    it.skip('clicking from the global search to the mega-menu', () => {
```

Comment:
What do you think about using `xit(…)` instead, as that's the syntax we use elsewhere?